integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -298,6 +304,54 @@ public void shouldModifyFetchMessage() throws Exception {
 
     }
 
+    @Test
+    public void proxySslClusterPlain() throws Exception {
+        String proxyAddress = "localhost:9192";
+
+        var certificateGenerator = new KeytoolCertificateGenerator();
+        certificateGenerator.generateSelfSignedCertificateEntry("test@redhat.com", "localhost", "KI", "RedHat", null, null, "US");
+
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder().testInfo(testInfo).build())) {
+            cluster.start();
+
+            String config = """
```

Comment:
Not for this PR but I wonder if we should look at some sort standardised config generation thing rather than having semi-random config docs floating around.